### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by mat-hek**

### CODE REVIEW

File: lib/membrane_caps_video_h264.ex [hunk 1]
```diff
@@ -56,6 +56,18 @@ defmodule Membrane.Caps.Video.H264 do
...
   @type alignment_t :: :au | :nal | :none
 
+  @typedoc """
+  When alignment is set to `:au`, determines whether buffers have NALu info attached in metadata.
+
+  If true, each buffer contains the NAL units list under `metadata.h264.nalus`. The list consists of
+  maps with the following entries:
+  - `prefixed_poslen: {pos, len}` - position and length of the NALu within the payload
+  - `unprefixed_poslen: {pos, len}` - as above, but omits Annex B prefix
+  - `metadata: metadata` - metadata that would be merged into the buffer metadata
```
Comment (mkaput) [inline lib/membrane_caps_video_h264.ex:66]: I think we might want to add an abstraction to express buffers that contain sub-buffers (in the future at least)

**Review by mkaput — APPROVED** [review]

File: lib/membrane_caps_video_h264.ex [hunk 2]
```diff
@@ -56,6 +56,18 @@ defmodule Membrane.Caps.Video.H264 do
   """
   @type alignment_t :: :au | :nal | :none
 
+  @typedoc """
+  When alignment is set to `:au`, determines whether buffers have NALu info attached in metadata.
+
+  If true, each buffer contains the NAL units list under `metadata.h264.nalus`. The list consists of
+  maps with the following entries:
+  - `prefixed_poslen: {pos, len}` - position and length of the NALu within the payload
+  - `unprefixed_poslen: {pos, len}` - as above, but omits Annex B prefix
```
Comment (mkaput) [inline lib/membrane_caps_video_h264.ex:65]: maybe `poslen` -> `span` ?

File: lib/membrane_caps_video_h264.ex (hunk 1 above)
Comment (mat-hek) [reply lib/membrane_caps_video_h264.ex:66]: I would consider that when it's needed in more places. We need to standardize metadata in the first place

File: lib/membrane_caps_video_h264.ex (hunk 2 above)
Comment (mat-hek) [reply lib/membrane_caps_video_h264.ex:65]: I wanted it to conform to https://erlang.org/doc/man/binary.html#part-2

File: lib/membrane_caps_video_h264.ex (hunk 1 above)
Comment (mkaput) [reply lib/membrane_caps_video_h264.ex:66]: Sure, so do I, but mentioning the idea :)

**Pull request merged**